static/icons/right-arrow.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="26" height="64" viewBox="0 0 26 64" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I would say let's use material icons as much as we can. In that way, we will be able to keep some homogenous UI through all the apps without adding extra assets. Material icon `navigate_next` might work... Or it looks too bad?

Reply:
Looks a little bit different but could work